platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformBuilder.java

```diff
@@ -147,15 +153,20 @@ public final class PlatformBuilder {
      * @param selfId              the ID of this node
      * @param softwareVersion     the software version of the application
      * @param initialState        the genesis state supplied by the application
```

Comment:
I realize this is not modified in this PR, however I believe the javadoc for this parameter and the method itself are wrong. The state provided may be a genesis state, or it may be a state loaded from disk which is not considered a genesis state. The method javadoc says that this builder tries to load an existing app state from a snapshot on disk, but that responsibility has been moved to services.